server/store/apistore/poll.go

```diff
@@ -0,0 +1,43 @@
+package apistore
```

Comment:
What does `api` of `apistore` represent?
I think that `pluginstore` or `kvstore` is better than `apistore`.

Reply:
I agree and changed it to `kvstore`.